generate-feedback/main.py

```diff
@@ -0,0 +1,175 @@
+import pickle
+import json
+import os
+import random
+
+# Macrouri pt limitele numarului de studenti
+MIN_STUDENTS = 20
+MAX_STUDENTS = 100
```

Comment:
Move this into a configuration file. Use the example from [config parsing for retrieving feedback](https://github.com/cs-pub-ro/feedback/blob/master/retrieve-feedback/retrieve_feedback.py#L36).